.gitignore

```diff
@@ -6,6 +6,8 @@ dunendggd.egg-info
 *~
 *#
 *.bak
+*.xml
+*plotting.py
```

Comment:
What is this `*plotting.py`? Should this really be ignored in the main repository? Same with the xml files. Are they a product of the workflow in the repository?